### ISSUE TITLE
How to calculate anchor scales and ratio's for custom dataset?

### ISSUE THREAD

**Issue by TheanMS** [NVIDIA/retinanet-examples#258]
Hi,
I want to train on custom dataset with 4 classes and every image contains 4 objects with A,B,C,D tags. Where the width to height ratio's of A,B,C,D are 2:1,8:1,12:1 and 3:1.
In 1st iteration, I used the default value of anchor ratio's  to train the model and the model was predicting only the object A and D.
Again in 2nd iteration, randomly changed the anchor ratio's to [0.5,0.125,2.0] for training, and the model was predicting few objects of tag A,B,C and D. And even I thought to train by increasing the number of anchor ratio's and anchor scales but I was getting error since I used pretrained weight. And I also trained with different input sizes, But the model results are too worst.

So I how can I change the anchor ratio's and scales to improve the model accuracy? Is there any calculation involved?

**Comment by yashnv** [comment]
No, this is a hyperparameter. You will have to try and comeup with thembby yourself. The ones we selected were considered good for this repository and COCO2017.
